test/common/cdp.py

```diff
@@ -137,10 +159,45 @@ def find_cdp_port(self):
 
     def get_browser_path(self):
         if self._browser_path is None:
-            self._browser_path = browser_path()
+            self._browser_path = browser_path(self.browser)
 
         return self._browser_path
 
+    def browser_cmd(self, cdp_port, env):
+        exe = self.get_browser_path()
+        if not exe:
+            raise SystemError(self.browser + " is not installed")
+
+        if self.browser == "chromium":
+            return [exe, "--headless", "--disable-gpu", "--no-sandbox", "--disable-setuid-sandbox",
+                    "--disable-namespace-sandbox", "--disable-seccomp-filter-sandbox",
+                    "--disable-sandbox-denial-logging", "--disable-pushstate-throttle",
+                    "--window-size=1920x1200", "--remote-debugging-port=%i" % cdp_port, "about:blank"]
+        elif self.browser == "firefox":
+            subprocess.Popen(["firefox", "--headless", "--no-remote", "-CreateProfile", "blank"], env=env).communicate()
+            p = subprocess.Popen("ls ~/.mozilla/firefox", shell=True, stdout=subprocess.PIPE, env=env)
+            in_profile = p.communicate()[0].decode("utf-8").split("\n")
+            userjs = "~/.mozilla/firefox/"
+            handler = "~/.mozilla/firefox/"
+            for item in in_profile:
+                if item.endswith("blank"):
+                    handler = os.path.join(handler, item, "handlers.json")
+                    userjs = os.path.join(userjs, item, "user.js")
+                    break
```

Comment:
Looks like you don't really need to read the whole directory in the first place? How is that different to simply doing `os.path.join(sef._browser_home, "blank")`? (And then tacking on the two *.json, of course).

Reply:
The path looks something like `~/.mozilla/firefox/ldsyl7au.blank/`. I could use `glob` though?

Reply:
Argh, sorry -- I misread the code. Yes, glob seems fine indeed.